views/sdtdServer/partials/settings/commandSettings.ejs

```diff
@@ -48,44 +50,87 @@
     <p class="form-text text-muted">
       CSMM's teleport system. Includes a suite of commands to set, remove, make public/private teleports.
     </p>
+    <div class="form-text" id="player-teleport-settings">
+      <hr>
+
+      <h4>Player teleport settings</h4>
+
+      <div class="form-group">
+        <label for="">Max locations that players can set</label>
+        <input type="text" class="form-control" name="inputs-settings-commands-playerteleports-maxlocations" id="inputs-settings-commands-playerteleports-maxlocations"
+          placeholder="<%= config.maxPlayerTeleportLocations %>">
+        <input name="inputs-settings-playerteleports-maxlocations-btn" id="inputs-settings-playerteleports-maxlocations-btn" class="btn btn-primary"
+          type="button" value="Set">
+      </div>
+
+      <div class="form-group">
+        <label for="">Time (in seconds) that a player has to wait before teleporting again</label>
+        <input type="text" class="form-control" name="inputs-settings-commands-playerteleports-timeout" id="inputs-settings-commands-playerteleports-timeout"
+          placeholder="<%= config.playerTeleportTimeout %>">
+        <input name="inputs-settings-playerteleports-timeout-btn" id="inputs-settings-playerteleports-timeout-btn" class="btn btn-primary"
+          type="button" value="Set">
+      </div>
+
+      <div class="form-group">
+        <label for="">Time (in seconds) that a teleport will be delayed. This makes sure players cannot use tele to escape a dire situation.</label>
+        <input type="text" class="form-control" name="inputs-settings-commands-playerteleports-delay" id="inputs-settings-commands-playerteleports-delay"
+          placeholder="<%= config.playerTeleportDelay %>">
+        <input name="inputs-settings-playerteleports-delay-btn" id="inputs-settings-playerteleports-delay-btn" class="btn btn-primary"
+          type="button" value="Set">
+      </div>
+    </div>
   </div>
 
+
+
+  <div class="form-check">
+    <label class="form-check-label">
+      <input type="checkbox" class="form-check-input" name="input-settings-commands-vote-checkbox" id="input-settings-commands-vote-checkbox"> Vote - lets players vote for your server directly In-Game
+    </label>
+    <p class="form-text text-muted">
+      CSMM's In-Game Voting system. Includes a suite of commands to set, serverKey and possible Currency Reward.
+    </p>
+        <div class="form-text" id="vote-for-server-settings">
+        <hr>
+
+        <h4>Vote for server settings</h4>
+
+        <div class="form-group">
+          <label for="">Your server API KEY for (7daystodie-servers.com)</label>
+          <p class="form-text text-muted">*You can add more than one server, by separating them with a semicolon ";" IMPORTANT: Note that the Currency reward will only be attributed for EACH server.</p>
```

Comment:
This text should be made more clear I think. 

This allows a player to vote on all servers of a host and claim multiple rewards on the server he/she plays on.

What if a player wants the reward individually for each server instead?

Reply:
Yes the text can be clearer and the one you suggest here is pretty clear.

Your question is valid but I think we should ask server hosts for that. My experience 'As a player' this feature would please me even more so that I could get more coins on the server I want them when voting directly In-Game. So if I did play on more than one server of the same Hosts.. like Corran's, I'd be voting from the server where I need the most coins and if I truly wanted them separately.. I'd go vote on the website. But I wouldn't play on 2 server at the same time anyways and of all the friends I have and ppl I know playing this game... no one has time to split on two different games.
But... that's just me so I can't answer for everyone.